Type mistmatch with Period.compare(dt: ICAL.Time | ICAL.Period) - `Cannot find namespace 'ICAL'.`

lib/ical/period.js:compare() produces:

```
../ical.js/dist/types/period.d.ts:126:17 - error TS2503: Cannot find namespace 'ICAL'.

126     compare(dt: ICAL.Time | ICAL.Period): 0 | 1 | -1;
                    ~~~~

../ical.js/dist/types/period.d.ts:126:29 - error TS2503: Cannot find namespace 'ICAL'.

126     compare(dt: ICAL.Time | ICAL.Period): 0 | 1 | -1;
                                ~~~~
```

```diff
index eb8044e..4f4acf4 100644
--- a/lib/ical/period.js
+++ b/lib/ical/period.js
@@ -223,7 +223,7 @@ class Period {
    * date is within the period, and -1 when the period is before the date. When comparing two
    * periods, as soon as they overlap in any way this will return 0.
    *
-   * @param {ICAL.Time|ICAL.Period} dt    The date or other period to compare with
+   * @param {Time|Period} dt    The date or other period to compare with
    */
   compare(dt) {
     if (dt.compare(this.start) < 0) {

```

--- a/lib/ical/period.js
+++ b/lib/ical/period.js
@@ -223,7 +223,7 @@ class Period {
    * date is within the period, and -1 when the period is before the date. When comparing two
    * periods, as soon as they overlap in any way this will return 0.
    *
-   * @param {ICAL.Time|ICAL.Period} dt    The date or other period to compare with
+   * @param {Time|Period} dt    The date or other period to compare with
    */
   compare(dt) {
     if (dt.compare(this.start) < 0) {

--- a/lib/ical/time.js
+++ b/lib/ical/time.js
@@ -1073,7 +1073,7 @@ class Time {
   /**
    * Compares the ICAL.Time instance with another one, or a period.
    *
-   * @param {ICAL.Time|ICAL.Period} aOther        The instance to compare with
+   * @param {Time|Period} aOther                  The instance to compare with
    * @return {Number}                             -1, 0 or 1 for less/equal/greater
    */
   compare(other) {

--- a/lib/ical/vcard_time.js
+++ b/lib/ical/vcard_time.js
@@ -21,7 +21,7 @@ import { pad2, strictParseInt } from "./helpers.js";
  * Also, normalization is not yet implemented for this class!
  *
  * @memberof ICAL
- * @extends {ICAL.Time}
+ * @extends {Time}
  */
 class VCardTime extends Time {
   /**

--- a/tools/jsdoc-collect-types.cjs
+++ b/tools/jsdoc-collect-types.cjs
@@ -22,6 +22,11 @@ exports.handlers = {
     if (doclet.kind === "typedef" && doclet.scope === "static" && doclet.longname.startsWith("ICAL.")) {
       typedefs.add({name: doclet.name, full: doclet.longname})
     }
+    if (doclet.kind === "class" && doclet.augments) {
+      for (let aug of doclet.augments) {
+        typedefs.add(aug)
+      }
+    }
   },
   processingComplete: function() {
     fs.writeFileSync("./tools/jsdoc-symbols-temp.json", JSON.stringify({gIcalClasses: Array.from(gIcalClasses), typedefs: Array.from(typedefs)}));

--- a/tools/jsdoc-ical.cjs
+++ b/tools/jsdoc-ical.cjs
@@ -55,8 +55,18 @@ exports.handlers = {
     ({ gIcalClasses, typedefs } = JSON.parse(fs.readFileSync("./tools/jsdoc-symbols-temp.json", "utf-8")));
   },
   newDoclet: function({ doclet }) {
+
     if (doclet.type) {
       augmentTypes(doclet.type);
+    }
+
+    if (doclet.augments) {
+      addPrefix(doclet.augments);
+    }
+
+    // Remove the duplicate design entry
+    if (doclet.longname == "ICAL.module:design" && doclet.comment == "/** @module ICAL.design */") {
+      doclet.undocumented = true;
     }
 
     if (doclet.returns) {

